arch/renesas/src/rx65n/rx65n_rtc.c

```diff
@@ -774,7 +774,6 @@ int up_rtc_settime(FAR const struct timespec *tp)
 
   /* Day of the week (0-6, 0=Sunday) */
 
-#if defined(CONFIG_LIBC_LOCALTIME) || defined(CONFIG_TIME_EXTENDED)
```

Comment:
Should this be just conditional on CONFIG_TIME_EXTENDED now?

Reply:
The CONFIG_TIME_EXTENDED is redundant after this commit:
```
commit 1e3f457c9ef71cbaa751d8ad8348ad1993b956b0
Author: Xiang Xiao <xiaoxiang@xiaomi.com>
Date:   Thu Jan 23 08:17:22 2020 -0600

    Remove TIME_EXTENDED option to more conform C standard
    
    Gregory Nutt <gnutt@nuttx.org>
    
        Run all .c and .h files modified in this PR through nxstyle and correct all coding standard problems.
    
    Xiang Xiao <xiaoxiang@xiaomi.com>
    
        Remove TIME_EXTENDED option to more conform C standard
    
        Note: the code/data size increment is small
```

Reply:
Thanks for clarifying.